### PR TITLE
gauche 0.9.8

### PR DESCRIPTION
**Pull request by tsahara**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by tsahara** [comment]
Hmm.  I found the issue has been fixed on master ( https://github.com/shirok/Gauche/pull/483 ).

File: Formula/gauche.rb [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+  url "https://downloads.sourceforge.net/gauche/Gauche/Gauche-0.9.8.tgz"
+  sha256 "3eb30d1051d8b48999fe46511c9f6983057735312c9832b7db13f9db140db74b"
 
   bottle do
     sha256 "c04f0c743c998cd8c63483c9f4d47311e9df465144390dc8d6db266499f9aaf5" => :mojave
     sha256 "7aa642136a7d5be56e21b2ffa424d0f0dbf09f1f4789733f2fd73bb551212ace" => :high_sierra
     sha256 "9aa6541d4840bfcb1c5b62a4f66c2c9a071cbeb03d7913798dc6e86261ea29f6" => :sierra
   end
 
+  patch do
```
Comment (Moisan) [inline Formula/gauche.rb]: Could you add comment pointing to the upstream issue?

**Review by Moisan — CHANGES_REQUESTED** [review]
Only needs a quick comment and squashing you commits into one.

File: Formula/gauche.rb (hunk 1 above)
Comment (tsahara) [reply Formula/gauche.rb]: Okay.  I added a comment and squashed commits.

**Comment by fxcoudert** [comment]
@brewtestbot test this please

**Comment by tsahara** [comment]
Thank you!

**Pull request closed without merging**